src/main/java/calculator/app/Main.java

```diff
@@ -1,4 +1,21 @@
 package calculator.app;
 
 public class Main {
+
```

Comment:
Main is not good place to put this code.
Imagine I'm your colleague and said "Hey, do you need Main functionality for your task?"

You will reply to me "What? What main functionality, what you are talking about".
How we can understand from class name - what class do?

Main is class that is entry point for execution program.

So please name class to Calculator, ArithmeticCalculator, BasicCalculator etc, but class name should be always correspond with its content